documentation/decisions/008-specify-quality-requirements.md

```diff
@@ -0,0 +1,22 @@
+# 008: Architects are able to specify quality requirements
+
+## Status: Accepted
+
+## Context 
+Within the FOUNDATION level working group we had long (aka _endless_) discussions about quality and quality requirements.
+Conclusion is:
+
+* Software architects shall be able to formulate precise quality requirements, i.e. as quality scenarios.
+* The shall not _invent_ these, but formulate in cooperation or collaboration with other stakeholders.
```

Comment:
Typo: Should be "They shall"

Reply:
my fault, I accidently mixed two completely different topics. We need to take the ADR-008 out of THIS discussion and PR. Sorry, @rhoadesre, seems I got confused yesterday...